images/libvirtd/libvirtd-ds.yaml

```diff
@@ -12,40 +12,21 @@ spec:
     spec:
       hostNetwork: true
       hostPID: true
+      hostIPC: true
       securityContext:
         runAsUser: 0
       containers:
-        - name: daemon
+        - name: libvirtd
           image: kubevirt/libvirtd:latest
           ports:
             - containerPort: 16509
           securityContext:
             privileged: true
             runAsUser: 0
           volumeMounts:
-            - mountPath: /etc/libvirt
-              name: etc
-            - mountPath: /var/run/libvirt
```

Comment:
No, /var/lib (like /etc) must stay inside the container.
We get permission issues. And it's essential for having libvirtd really isolated.